atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/AbstractNodeAvailabilityTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.cassandra.multinode;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import com.palantir.atlasdb.keyvalue.api.KeyValueService;
+import com.palantir.atlasdb.keyvalue.api.NodeAvailabilityStatus;
+
+public abstract class AbstractNodeAvailabilityTest {
```

Comment:
A parametrized class will run with, say, 3 sets of parameters under the same setup. This means one node down will run `NO_QUORUM_AVAILABLE` and `LOCAL_QUORUM_AVAILABLE`. This is not the intent here we want one node down to run `LOCAL_QUORUM_AVAILABLE` and 2/3 nodes down to only run `NO_QUORUM_AVAILABLE`.